yolapy/models/site.py

```diff
@@ -14,4 +14,12 @@ def __init__(self, **kwargs):
     def get(cls, site_id):
         """Get a site from the Yola API."""
         site_attributes = Yola().get_site(site_id)
-        return Site(**site_attributes)
+        return cls(**site_attributes)
+
+    @classmethod
+    def for_user(cls, user_id):
```

Comment:
...or I think this might be better:

We've already established the concept of `list` for a model: https://github.com/yola/yolapy/blob/98118cbcf42362e107caaa7dc5d8350168c9db47/yolapy/models/subscription.py#L52

so if we add that to `Site`, getting a user's site would look like this:

```python
Site.list(user_id=user_id)[0]
```

which pushes the knowledge of (one site per user) from this client package down into the consumer. I think it makes more sense there anyway.

You're not around but I'm going to make this change anyway. Maybe @RayeN could take a look.